types/office-js/index.d.ts

```diff
@@ -8855,9 +8855,11 @@ declare namespace Office {
             /**
              * The attachment is stored in a cloud location, such as OneDrive.
              *
-             * **Important**: In Read mode, the `id` property of the attachment's {@link Office.AttachmentDetails | details} object
+             * **Important**: In Read mode, the `id` property of the attachment's
+             * {@link https://learn.microsoft.com/javascript/api/outlook/office.attachmentdetails | AttachmentDetails} object
              * contains a URL to the file.
-             * From requirement set 1.8, the `url` property included in the attachment's {@link Office.AttachmentDetailsCompose | details} object
+             * From requirement set 1.8, the `url` property included in the attachment's
+             * {@link https://learn.microsoft.com/javascript/api/outlook/office.attachmentdetails | AttachmentDetails} object
```

Comment:
Should this still point to AttachmentDetailsCompose?

```suggestion
             * {@link https://learn.microsoft.com/javascript/api/outlook/office.attachmentdetailscompose | AttachmentDetailsCompose} object
```